types/setting.go

```diff
@@ -966,6 +992,15 @@ func ValidateSetting(name, value string) (err error) {
 			return fmt.Errorf("the value of %v is invalid: %v", sName, err)
 		}
 
+	case SettingNameRecurringSuccessfulJobsHistoryLimit:
+		if _, err := strconv.Atoi(value); err != nil {
+			return fmt.Errorf("value %v is not a number", value)
```

Comment:
Why not use `errors.Wrapf`?

Reply:
Ray, need to align with others at least to have err in the message. 

I think we should refactor returned errors for each setting in this method later but use wrap instead to embed the original error if needed.

Reply:
I have done to wrap the error to return.
https://github.com/longhorn/longhorn-manager/blob/a9c3853113e0d70b114260aeeaba020f3f2d9240/types/setting.go#L995-L1002

And I also modified others' return errors as wrapping errors message.